src/components/Login.tsx

```diff
@@ -182,7 +182,7 @@ class Component extends React.Component<any, any> {
               ? 'Or login with one of the following services'
               : 'Login with one of the following'}
           </h3>
-          {providers.map(({ name, path, Icon }) => {
+          {providers.filter(provider => !(provider.name === LoginProvider.Passport && !PASSPORT_ENABLED)).map(({ name, path, Icon }) => {
```

Comment:
Hard to interpret this but I believe the logic is "remove Passport type progivders if passport is not enabled"...

Can we simplify it to "all true when passport_enabled is true, otherwise not the passport providers":

```suggestion
          {providers.filter(provider => PASSPORT_ENABLED || provider.name !== LoginProvider.Passport).map(({ name, path, Icon }) => {
```

Reply:
changed as suggested